doc/api/http2.md

```diff
@@ -1147,7 +1147,8 @@ of the given file:
 
 If an error occurs while attempting to read the file data, the `Http2Stream`
 will be closed using an `RST_STREAM` frame using the standard `INTERNAL_ERROR`
-code.
+code. The `onError` callback will be called before send if is defined, otherwise
```

Comment:
The `send if is defined` sounds somewhat wrong to me. Can we change that to something like
``If the `onError` callback is defined it will be called, otherwise the stream will be destroyed.``?